vsutil/__init__.py

```diff
@@ -251,6 +251,65 @@ def is_image(filename: str, /) -> bool:
     return types_map.get(path.splitext(filename)[-1], '').startswith('image/')
 
 
+def scale(value: float, 
```

Comment:
Will typecheckers complain about this not being Union[float, int]`?

Reply:
I guess I might as well change it just to be safe